src/test/java/software/amazon/awssdk/crt/test/MqttClientConnectionFixture.java

```diff
@@ -306,13 +320,16 @@ void checkOperationStatistics(
                 fail("Incomplete operations size:" + incomplete_ops_size + " did not equal expected value:" + expectedIncompleteOperationSize);
             }
 
-            if (unacked_ops_count != expectedUnackedOperationCount) {
-                fail("Unacked operations count:" + unacked_ops_count + " did not equal expected value:" + expectedUnackedOperationCount);
-            }
+            // While in theory we should be checking the unacked operations as well, it isn't really straight forward
```

Comment:
Do we want to change this function and remove the two arguments that aren't being actually checked (or anything done to them at all) for now and leave a TODO so that if you're looking at the tests themselves you're not giving it these two members expecting something is being done from that perspective?